src/bootstrap.ts

```diff
@@ -49,6 +49,32 @@ export const bootstrap = async () => {
 
   // Health check
   app.get('/healthcheck', (_, res) => res.sendStatus(200))
+  app.get('/hc', function (_, res) {
+    res.redirect('/healthcheck')
+  })
+
+  // Diagnostics
+  app.get('/diag', authMiddleware(true), (_, res) => {
+    config.gsuiteScopes.then((value) => {
+      const routes = []
+      app._router.stack.forEach(function (route) {
+        if (route.route && route.route.path) {
+          routes.push(route.route.path)
+        }
+      })
+
+      res.json({
+        'tools': process.versions,
+        'version': process.env.npm_package_version,
+        'routes': routes,
+        'gcp_key_vault_is_set': Boolean(process.env.GCP_SECRET_MANAGER_PROJECT_ID),
```

Comment:
This is useless without calling the key vault.

Reply:
that the thing... you can set the key vault name, but not the credentials. Endpoint will return true, but keyvault is not configured properly => it's false positive